src/main/java/com/celements/web/FileAction.java

```diff
@@ -312,8 +317,11 @@ public boolean renderFileFromObjectField(String filename, XWikiDocument doc,
   public boolean renderFileFromAttachment(String filename, XWikiDocument doc,
       XWikiContext context) throws IOException, XWikiException {
     _Logger.debug("... as attachment");
-    XWikiAttachment attachment = doc.getAttachment(filename);
-    if (attachment != null) {
+    try {
+      IAttFileNameMatcherRole matcher = Utils.getComponent(IAttFileNameMatcherRole.class);
```

Comment:
this will result in an ComponentLookup Exception, because there is no "default" component.